server.js

```diff
@@ -1,29 +1,131 @@
-import express from "express";
-import cors from "cors";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
+import express, { request } from "express"; // to create api
+import cors from "cors"; //cors- To have request from same origin. Frontend and backend can both work from localhost
+import netflixData from "./data/netflix-titles.json";
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
+
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
-app.use(express.json());
+app.use(express.json()); //Allows us to read the body from the request as a json
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const navigation = {
+    guide: "Routes for this netflix API!",
+    Endpoints: [
+      {
+        "/netflix": "Display all netflix data",
```

Comment:
In order to follow RESTful naming conventions, I'd suggest changing the name of your netflix route to one in plural form, netflix-titles maybe? Same thing for type → types